src/Symfony/Component/Console/Tester/ApplicationTester.php

```diff
@@ -96,6 +121,30 @@ public function getDisplay($normalize = false)
     }
 
     /**
+     * Gets the output written to STDERR by the application.
+     *
+     * @param bool $normalize Whether to normalize end of lines to \n or not
+     *
+     * @return string
+     */
+    public function getErrorOutput($normalize = false)
+    {
+        if (!$this->captureOutputStreamsIndependent) {
+            throw new \LogicException('Error output not available when tester run without "capture_stderr_separately" option set.');
```

Comment:
:+1:   updated, I've rename the variable (https://github.com/symfony/symfony/pull/17255#discussion_r55712019) as well